docs/docs-source/docs/modules/get-started-gcp-marketplace/pages/deploy-cloudflow.adoc

```diff
@@ -0,0 +1,36 @@
+= Deploy Cloudflow on GCP Marketplace
+:toc:
+:toc-title: ON THIS PAGE
+:toclevels: 2
+
+== Deploying Cloudflow
+
+You have followed the xref:find-cloudflow-on-gcp-marketplace.adoc[previous step] of locating Cloudflow in the Marketplace catalog and now see the following screen:
+
+.The Cloudflow page on Marketplace
+image::cloudflow-main-gcp.png[Cloudflow page on Marketplace]
+
+Spend some time reading through the information on the page, especially the section on pricing. When you are ready, click "Configure" to proceed to the deployment screen:
```

Comment:
Why do we want to highlight the pricing? This is more a question for our product gurus out there (@longmuir @kwehden)

Reply:
We don't. Our pricing is made very clear in big font, no need to draw attention to it.

Reply:
Ok. I'll remove that part.